eth/stagedsync/stage_issuance.go

```diff
@@ -53,20 +55,15 @@ func SpawnStageIssuance(cfg IssuanceCfg, s *StageState, tx kv.RwTx, ctx context.
 		return fmt.Errorf("getting headers progress: %w", err)
 	}
 
-	if headNumber == s.BlockNumber {
-		return nil
-	}
-	if cfg.chainConfig.Consensus != params.EtHashConsensus {
-		if err = s.Update(tx, headNumber); err != nil {
-			return err
-		}
+	if cfg.chainConfig.Consensus != params.EtHashConsensus || !cfg.enabledIssuance || headNumber == s.BlockNumber {
```

Comment:
1. why we using 2 names for same thing? "issuance" and "watchTheBurn"
2. let's add some section in Readme.md about optional stages and put new cli flag there.

Reply:
issuance is the stage name within the code, UX-wise me and Alexey decided to call it watchTheBurn